presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.245
+=============
+
+.. warning::
+    There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
+
+**Highlights**
+==============
+* New :func:`enum_key` UDF to get the key for an enum value.
```

Comment:
I ran make with Sphinx 2.4.0 and it gives me warning about this line. 
```
WARNING: py:func reference target not found: enum_key  
```

We either need to document the function, or do
```
``enum_key``
```
as a temporary workaround to not reference the function.